pkg/loop/internal/relayer/pluginprovider/chainreader/test/chain_reader_loop_tester.go

```diff
@@ -8,33 +8,73 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb"
 	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/relayer/pluginprovider/chainreader"
 	"github.com/smartcontractkit/chainlink-common/pkg/types"
+
 	. "github.com/smartcontractkit/chainlink-common/pkg/types/interfacetests" //nolint common practice to import test mods with .
 )
 
+func LoopEncodingTestMatrix(t *testing.T, test func(chainreader.EncodingVersion) func(t *testing.T)) {
```

Comment:
Probably no need to say LOOP given the package scope:
```suggestion
func EncodingTestMatrix(t *testing.T, test func(chainreader.EncodingVersion) func(t *testing.T)) {
```